docs/src/content/docs/fr/getting-started.mdx

```diff
@@ -89,4 +126,4 @@ Utilisez la liste croissante de guides dans la barre latérale pour vous aider 
 
 Si vous ne trouvez pas votre réponse dans cette documentation, veuillez consulter la [documentation complète d'Astro](https://docs.astro.build). Votre question peut être résolue en comprenant comment Astro fonctionne en général, sous ce thème Starlight.
 
-Vous pouvez également obtenir de l'aide dans le [Discord Astro](https://astro.build/chat/) de notre communauté active et sympathique ! Vous pouvez poster des questions dans notre forum `#support` ou visiter notre canal dédié `#starlight` pour discuter des développements en cours et plus encore !
+Vous pouvez également vérifier s'il existe des [problèmes connus de Starlight sur GitHub](https://github.com/withastro/starlight/issues) et obtenir de l'aide dans le [Discord Astro](https://astro.build/chat/) de notre communauté active et sympathique ! Vous pouvez poster des questions dans notre forum `#support` avec le tag "starlight" ou visiter notre canal dédié `#starlight` pour discuter des développements en cours et plus encore !
```

Comment:
Reference for the "post" translation https://www.culture.fr/franceterme/terme/INFO738

```suggestion
Vous pouvez également vérifier [tous les problèmes connus de Starlight sur GitHub](https://github.com/withastro/starlight/issues) et obtenir de l'aide dans le [Discord Astro](https://astro.build/chat/) de notre communauté active et sympathique ! Publiez des questions dans notre forum `#support` avec le tag "starlight" ou visiter notre canal dédié `#starlight` pour discuter des développements en cours et plus encore !
```